src/main.ts

```diff
@@ -43,17 +44,17 @@ async function start () {
   // must come first as inits below depend on config
   await fetchConfig()
 
-  if (window['sentryEnable'] === 'true') {
+  if ((window as any).sentryEnable === 'true') {
```

Comment:
ES Lint wanted to change this to `window.sentryEnable` but the window object doesn't have that property and this caused an error.

The correct to fix this is to extend the window object to declare any additional properties ... I'm calling that out of scope for this cleanup.